pkg/segment/structs/segstructs.go

```diff
@@ -239,6 +239,24 @@ type LetColumnsRequest struct {
 	SortColRequest       *SortExpr
 	NewColName           string
 	MultiValueColRequest *MultiValueColLetRequest
+	FormatResults        *FormatResultsRequest // formats the results into a single result and places that result into a new field called search.
+}
+
+// formats the results into a single result and places that result into a new field called search.
+type FormatResultsRequest struct {
+	MVSeparator   string         // separator for multi-value fields. Default= "OR"
+	MaxResults    uint64         // max number of results to return
+	EmptyString   string         // string to return if no results are found. Default= "NOT()"
+	RowColOptions *RowColOptions // options for row column
+}
+
+type RowColOptions struct {
+	RowPrefix       string // prefix for row column. Default= "("
+	ColumnPrefix    string // prefix for column column. Default= "("
+	ColumnSeparator string // separator for column column. Default= "AND"
+	ColumnEnd       string // end for column column. Default= ")"
+	RowSeparator    string // separator for row column. Default= "OR"
+	RowEnd          string // end for row column. Default= ")"
 }
```

Comment:
I'm not sure what you mean by `row column` and `column column`. Are these typos?